StructureDefinition/NHSDigital_FamilyMemberHistory_Genomics.StructureDefinition.json

```diff
@@ -0,0 +1,97 @@
+{
+  "resourceType": "StructureDefinition",
+  "url": "https://fhir.nhs.uk/StructureDefinition/NHSDigital_FamilyMemberHistory_Genomics",
+  "name": "NHSDigital_FamilyMemberHistory_Genomics",
+  "status": "draft",
+  "date": "2023-02-06",
+  "fhirVersion": "4.0.1",
+  "kind": "resource",
+  "abstract": false,
+  "type": "FamilyMemberHistory",
+  "baseDefinition": "https://fhir.hl7.org.uk/StructureDefinition/UKCore-FamilyMemberHistory",
+  "derivation": "constraint",
+  "differential": {
+    "element": [
+      {
+        "id": "FamilyMemberHistory.identifier",
+        "path": "FamilyMemberHistory.identifier",
+        "min": 1,
+        "max": "1"
+      },
+      {
+        "id": "FamilyMemberHistory.identifier.assigner",
+        "path": "FamilyMemberHistory.identifier.assigner",
+        "type": [
+          {
+            "code": "Reference",
+            "targetProfile": [
+              "https://fhir.nhs.uk/StructureDefinition/NHSDigital-Organization-Genomics"
+            ]
+          }
+        ]
+      },
+      {
+        "id": "FamilyMemberHistory.patient",
+        "path": "FamilyMemberHistory.patient",
+        "type": [
+          {
+            "code": "Reference",
+            "targetProfile": [
+              "https://fhir.nhs.uk/StructureDefinition/NHSDigital-Patient-Genomics"
+            ]
+          }
+        ]
+      },
+      {
+        "id": "FamilyMemberHistory.age[x]",
```

Comment:
@KazeemHamzat, just questioning why age, estimatedAge and deceased have been profiled out of the resource (as well as making the identifier mandatory).

Reply:
@KazeemHamzat, As discussed with the GU, please update the cardinalities for the 3 fields identified. The FMH profile should be good once this is addressed